src/ObjectPool/test/DependencyInjectionExtensionsTest.cs

```diff
@@ -0,0 +1,312 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.ObjectPool;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.ObjectPool.TestResources;
+using Xunit;
+using System.Linq;
+using AlwaysTestTests;
+
+namespace Microsoft.Extensions.ObjectPool;
+
+public class DependencyInjectionExtensionsTest
+{
+   
+    [Fact]
+    public void ConfiguresPoolOptions()
+    {
+        var services = new ServiceCollection()
+            .Configure<PoolOptions>(typeof(TestClass).FullName, options => options.Capacity = 2048)
+            .Configure<PoolOptions>(typeof(TestDependency).FullName, options => options.Capacity = 4096)
+            ;
+        using var provider = services.BuildServiceProvider(validateScopes: true);
+
+        var sut = provider.GetRequiredService<IOptionsMonitor<PoolOptions>>();
```

Comment:
What does `sut` stand for?

Reply:
That doesn't seem like typical naming in the repo. I haven't seen it anywhere in runtime or aspnetcore.

Reply:
I can't comment on that, but I see it a lot "in the wild" where that's what it means.